src/data/IdentifierSelector.ts

```diff
@@ -1,14 +1,18 @@
 export type IdentifierSelector = {
-  accession?: string;
-  strain?: string;
-  sraAccession?: string;
+  accession?: string[];
+  strain?: string[];
+  sraAccession?: string[];
 };
 
 export function addIdentifierSelectorToUrlSearchParam(selector: IdentifierSelector, params: URLSearchParams) {
+  let commaSeparatedStr = '';
   for (const k of ['accession', 'strain', 'sraAccession'] as const) {
     const value = selector[k];
     if (value !== undefined) {
-      params.set(k, value);
+      for (let i of value) {
```

Comment:
You can use `join()` to create a comma-separated list ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join))